src/Root.tsx

```diff
@@ -1,14 +1,18 @@
 import { HashRouter as Router, Route, Routes } from 'react-router-dom';
 
-import App from './App';
 import { Paths, DeviceCategory } from './enums';
+
 import { CartPage, ProductsCatalog, NotFoundPage, ProductPage } from './pages';
+import { HomePage } from './pages/HomePage';
```

Comment:
You can add HomePage to index.ts & move HomePage to the fifth line. Will be something like this:
`import { CartPage, ProductsCatalog, NotFoundPage, ProductPage, HomePage } from './pages';`